lua/gitlinker/commons/spawn.lua

```diff
@@ -1,3 +1,10 @@
+local _system
```

Comment:
please don't manual edit the 'gitlinker.commons.*' modules, they're auto installed via github actions. see: https://github.com/linrongbin16/gitlinker.nvim/blob/1d5947143fac683eb36bbe0d7408620ad6102598/.github/workflows/ci.yml#L42-L54

Reply:
Oops, can you revert it pelease? I'm not in my computer right now. I remember having errors about the vim loop because of the usage of vim.fn (for some reason the error doesn't show directly, the code just doesn't run when the error is reached).

Reply:
got it, then it should be allowed, I will fix it in the commons.